packages/opentelemetry-resource-util/src/index.ts

```diff
@@ -197,7 +197,7 @@ export interface MonitoredResource {
  * @returns the corresponding GCM MonitoredResource
  */
 export function mapOtelResourceToMonitoredResource(
-  resource: Resource,
+  resource: IResource,
```

Comment:
The type was widened to the interface on the calling side of this function which was technically a breaking change for TypeScript. Since we use type inference I only have to correct here.